# HG changeset patch
# User Andreas Weber <octave@josoansi.de>
# Date 1546450593 -3600
#      Wed Jan 02 18:36:33 2019 +0100
# Branch stable
# Node ID b66765e66e8729ff6f59947ed253fa7d1ee7bb35
# Parent  45d5a4ae636b1f3f6c3ede460f38147457af1cd6
ov-fcn-inline.cc: Fix static analyzer detected V601 issue (bug #55347)

diff --git a/libinterp/octave-value/ov-fcn-inline.cc b/libinterp/octave-value/ov-fcn-inline.cc
--- a/libinterp/octave-value/ov-fcn-inline.cc
+++ b/libinterp/octave-value/ov-fcn-inline.cc
@@ -795,17 +795,17 @@ functions from strings is through the us
         fargs.append (std::string ("x"));
 
     }
   else if (nargin == 2 && args(1).isnumeric ())
     {
       if (! args(1).is_scalar_type ())
         error ("inline: N must be an integer");
 
-      int n = args(1).int_value ("inline: N must be an integer");
+      int n = args(1).xint_value ("inline: N must be an integer");
 
       if (n < 0)
         error ("inline: N must be a positive integer or zero");
 
       fargs.resize (n+1);
 
       fargs(0) = "x";
 
